app/jobs/release_object_job.rb

```diff
@@ -20,7 +20,7 @@ def perform(bulk_action_id, params)
     @manage_release = params[:manage_release]
     @webauth = OpenStruct.new params[:webauth]
     @pids = params[:pids]
-    File.open(bulk_action.log_name, 'w') do |log|
+    with_bulk_action_log do |log|
```

Comment:
inclined to leave this, since i didn't change the line count, and i think i cleaned it up slightly overall.